examples/energy-monitor/ui/desktop_window.slint

```diff
@@ -28,6 +28,8 @@ export component MainWindow inherits Window {
     min-width: 320px;
     min-height: 240px;
     background: Theme.palette.pure-black;
+    preferred-width: 900px;
+    preferred-height: 700px;
```

Comment:
I suggest the dimension from the Figma design.

```suggestion
    preferred-width: 800px;
    preferred-height: 480px;
```

That's a whole lot closer to the golden ratio and a common size.